fb.h

```diff
@@ -1,10 +1,25 @@
 #ifndef VCAM_FB_H
 #define VCAM_FB_H
 
+#include <linux/fb.h>
+
 #include "device.h"
 
 struct proc_dir_entry *init_framebuffer(const char *proc_fname,
                                         struct vcam_device *dev);
 void destroy_framebuffer(const char *proc_fname);
 
+int init_vcamfb(struct vcam_device *dev);
```

Comment:
You have to distinguish from `init_framebuffer` and `init_vcamfb`. I would like to simplify the interface.

Reply:
We can now refactor the name scheme of helper functions and make them more consistent. That is,
* `vcamfb_init`
* `vcamfb_destroy`
* `vcamfb_update`
* `vcamfb_get_name`